maintainer-guide.md

```diff
@@ -104,7 +104,8 @@ New releases are made every set period of time (typically every week), in which
 * Before release day:
   * [Create an issue for the release](https://github.com/TEAMMATES/teammates/issues/new?template=release.md) to announce the scheduled release time.
   * Update the "about page" with the names of new contributors, if any.
-  * Check if any schema change is needed for this release. If so, inform PM that this release requires maintainance mode and generate Liquibase changelog (refer to [Schema Migration](#schema-migration)).
+  * Check if any schema change is needed for this release. If so, generate Liquibase changelog (refer to [Schema Migration](#schema-migration)) and inform PM that this release requires maintainance mode.
```

Comment:
Is maintenance mode needed? I thought that prod should never go down for schema changes, we could correct this here

Reply:
Hopefully, most schema changes are backward compatible (i.e., no need to go into maintenance mode) but some might not be. So, should cover both cases in the docs.

Reply:
@damithc schema changes + data migration in SQL is no different than that in Datastore. There is no need for any additional concern.